addons/im_livechat/static/tests/helpers/mock_server/controllers/main.js

```diff
@@ -96,19 +96,28 @@ patch(MockServer.prototype, {
                 ["id", "=", channelVals.livechat_operator_id],
             ]);
             return {
+                id: -1,
```

Comment:
Ok for now. As I said in previous comment there's ongoing work in internal of discuss models that will help not having to forge such an id.
I think there's no rush to update it ASAP, as this implementation should work until we make further functional changes.